pyccel/codegen/utilities.py

```diff
@@ -70,7 +70,13 @@ def construct_flags(compiler,
 
     if accelerator is not None:
         if accelerator == "openmp":
-            flags += " -fopenmp"
+            if sys.platform == "darwin":
+                if compiler == "gcc":
+                    flags += " -Xpreprocessor -fopenmp -lomp"
```

Comment:
`-lomp` is a library. It shouldn't be added to the flags (see issue #489). Maybe we should add libs as an argument to this function?